docs/source/markdown/podman-create.1.md

```diff
@@ -858,16 +859,20 @@ named volume. Host paths are allowed to be absolute or relative; relative paths
 are resolved relative to the directory Podman is run in. Any source that does
 not begin with a `.` or `/` it will be treated as the name of a named volume.
 If a volume with that name does not exist, it will be created. Volumes created
-with names are not anonymous and are not removed by `--rm` and
-`podman rm --volumes`.
+with names are not anonymous. They are not removed by `--rm` option and
```

Comment:
remove by "the" ... and "the" ... command.

Reply:
Fixed